### DISCUSSION TITLE
Style helper plugin

### DISCUSSION THREAD

**Discussion by emsquid** [sayanarijit/xplr#555] [Show and tell]
Hey! I made a quite simple plugin, but also quite useful for those who like colors like me :art:

Basically it gives the possibility to somewhat _interact_ with the `Style` concept, which can be useful when writing plugins that may need to display files/folders in a non-default panel, or anything else I can't think about...

If people want to use it, or contribute to it (could be cool), here it is: [style.xplr](https://github.com/emsquid/style.xplr)

**Comment by sayanarijit** [comment]
Nice. At first I thought it uses xplr's [Style](https://xplr.dev/en/style) lua api, so I was thinking how it could be useful. But later found it generates ansi escape codes itself, and doesn't depend on any xplr api, so it's can be used anywhere, but the names are similar to xplr's Style api. You might want to update the docs to avoid the confusion.

Cool thing. Feel free to add it here: https://xplr.dev/en/awesome-plugins
